task/git-clone-oci-ta/0.1/git-clone-oci-ta.yaml

```diff
@@ -236,6 +238,7 @@ spec:
         fi
         printf "%s" "${RESULT_SHA}" >"$(results.commit.path)"
         printf "%s" "${PARAM_URL}" >"$(results.url.path)"
+        git log -1 --pretty=%ct >"$(results.commit-timestamp.path)"
```

Comment:
Let's avoid the trailing newline (like in https://github.com/konflux-ci/build-definitions/pull/1040, for example)
```
git log -1 --pretty=%ct | hexdump -c
0000000   1   7   1   8   2   9   1   6   1   7  \n
```

Reply:
updated